### PR TITLE
feat(onyx-1113): sell flow - additional documents step

### PR DESCRIPTION
**Pull request by nickskalkin**
The type of this PR is: **Feat**

This PR solves [ONYX-1113]

### Description

Made some assumptions how thumbnails should look like and behave, once it's on staging I'll ask Barney what needs to be changed.

image

[ONYX-1113]: https://artsyproduct.atlassian.net/browse/ONYX-1113?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/__tests__/UploadPhotos.jest.tsx
Comment (olerichter00) [inline src/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/__tests__/UploadPhotos.jest.tsx:477]: Should we have a more extensive list of supported files that we can use here? (like https://github.com/artsy/force/pull/14233/files#diff-a05481bf8da07c405652aacab0976bcc614c5748277025c7eca1b1fe0f96152fR133)

File: src/Apps/Sell/Routes/AdditionalRoutes/AdditionalDocumentsRoute.tsx
Comment (olerichter00) [inline src/Apps/Sell/Routes/AdditionalRoutes/AdditionalDocumentsRoute.tsx:68]: Could we use `setLoading` from the context here directly instead of passing it as a prop to `SubmissionLayout`? Same for`PhotosRoute.tsx`.

File: src/Components/PhotoUpload/Components/PhotoDropzone.tsx
Comment (olerichter00) [inline src/Components/PhotoUpload/Components/PhotoDropzone.tsx:17]: https://jsdoc.app/tags-deprecated

```suggestion
/**
 * @deprecated Deprecated - prefer using FileDropzone, which is a more generic version of this component
 * Also probably safe to remove this component entirely when cleaning up old sell flow
 */
```

**Review by olerichter00 — DISMISSED** [review]
Looks great! 🌟

**Review by olerichter00 — APPROVED** [review]

**Review by MounirDhahri — COMMENTED** [review]
Nice!

**Comment by MounirDhahri** [comment]
Need to mention it after a second review - great PR @nickskalkin  👏

**Pull request merged**
